pkg/controller/common.go

```diff
@@ -1259,3 +1278,152 @@ func containsString(key string, items []string) bool {
 	}
 	return false
 }
+
+type ReplicaRequest struct {
+	VolumeName string
+	Priority   int //The number of replicas that have yet to be created
+}
+type VolumeReplicaRequestsPriorityQueue struct {
+	mut   *sync.Mutex
+	queue *cache.Heap
+}
+
+func (vq VolumeReplicaRequestsPriorityQueue) Push(replicaRequest *ReplicaRequest) {
```

Comment:
The receiver parameter needs to be a pointer here, i.e. `vq *VolumeReplicaRequestsPriorityQueue`. As implemented, this function updates a local copy of the `VolumeReplicaRequestsPriorityQueue`, so it has no effect on the target object.